### DISCUSSION TITLE
Icon request: icon-footprint

### DISCUSSION THREAD

**Discussion by mennows** [FortAwesome/Font-Awesome#21147] [Icon Requests]
Hello,

I would like to reqeust a new icon.
![footprint](https://f.cloud.github.com/assets/5064719/1314382/ac306caa-3272-11e3-993f-1250eda28d19.jpg)

**Comment by JonnyHardwick** [comment]
+1

**Comment by Tsertse** [comment]
+1

**Comment by devilabit** [comment]
+1 useful for carbon footprint etc

**Comment by ggiakas** [comment]
+1

**Comment by kvanderd** [comment]
+1

**Comment by ghost** [comment]
+1

**Comment by jjsiddall** [comment]
+1

**Comment by Valesco** [comment]
+1!

**Comment by rodrigodesousa03** [comment]
+1

**Comment by xinwo** [comment]
+1

**Comment by doncadavona** [comment]
+1

**Comment by AndreLZGava** [comment]
Hello guys

I love font awesome, but some icons that we need take a lot of time to be added to font awesome, so I decided to create an extenssion to make it easy , if you would like to use them , you can download this at https://github.com/AndreLGava/font-awesome-extenssion. fa-footprint was an icon that i used. If there are any problem please create an issue on the GitHub project

**Comment by marshmn** [comment]
+1

**Comment by gcorderoflorin** [comment]
+1

**Comment by kpingel** [comment]
👍

**Comment by rodrigobranas** [comment]
There is an icon, fa-paw, which is quit similar context, although came from an animal...

http://fontawesome.io/icon/paw/

**Comment by akosphp** [comment]
+1

**Comment by iluhin** [comment]
yes!!!! pls add this icon!! single foot

**Comment by mccrearyk** [comment]
+1

**Comment by virmana** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
